docs/tutorials/fundamentals/viewer.md

```diff
@@ -404,40 +379,53 @@ viewer.camera.zoom = 2
 viewer.camera.angles = (3, 38, 53)
 nbscreenshot(viewer, alt_text="A rotated 3D view")
 ```
+![image: Rotated image](../assets/tutorials/rotated-image.png)  
 
-Next to the 2D / 3D button is a button to roll the dimensions that are currently
-being displayed in the viewer - for example if you have a `ZYX` volume and are
-looking at the `YX` slice this will then show you the `ZY` slice.
+The third button rolls the dimensions that are currently displayed in the viewer. For example if you have a `ZYX` volume and are looking at the `YX` slice, this will then show you the `ZY` slice.
 
-After that is a button that transposes the displayed dimensions.
+The fourth button transposes the displayed dimensions.
 
-After that is a button the toggles grid mode.
+Then there is a grid button that toggles grid mode. When clicked it displays each layer of the image in its own tile. 
```

Comment:
```suggestion
Then there is a grid button that toggles grid mode. When clicked it displays each layer of the image in its own tile. You can right-click this button to adjust the way the tiles are presented, such as the grid dimensions, the order of the layers in the tiles, and whether layers are overlayed in the tiles.
```

Reply:
Bump, I think this is still important to add here because it's not very discoverable.